discord/ext/commands/help.py

```diff
@@ -162,8 +169,8 @@ def pages(self):
         return self._pages
 
     def __repr__(self):
-        fmt = '<Paginator prefix: {0.prefix} suffix: {0.suffix} max_size: {0.max_size} count: {0._count}>'
-        return fmt.format(self)
+        fmt = '<Paginator prefix: {0.prefix} suffix: {0.suffix} linesep: {1} max_size: {0.max_size} count: {0._count}>'
+        return fmt.format(self, repr(self.linesep))
```

Comment:
```suggestion
        fmt = '<Paginator prefix: {0.prefix} suffix: {0.suffix} linesep: {0.linesep!r} max_size: {0.max_size} count: {0._count}>'
        return fmt.format(self)
```

Reply:
Might as well repr the prefix and suffix too while I'm at it.